JS/Lesson24/sinan/map.js

```diff
@@ -20,15 +20,15 @@ const fruits = wordsExercise1.map(word => {
 
 	return word.toUpperCase()
 });
-console.log(fruits);
+console.log('Exercise 1:',fruits);
```

Comment:
Changes in this file are not related to the Lesson22 homework, please revert (make sure that the code in map.js file is exactly as it is in the main branch in the same file right now).